src/NPCManager.cpp

```diff
@@ -54,11 +54,11 @@ void NPCManager::init() {
         for (nlohmann::json::iterator npc = npcData.begin(); npc != npcData.end(); npc++) {
             BaseNPC tmp(npc.value()["iX"], npc.value()["iY"], npc.value()["iZ"], npc.value()["iNPCType"],
                 npc.value()["iHP"], npc.value()["iConditionBitFlag"], npc.value()["iAngle"], npc.value()["iBarkerType"]);
-            
+
             // Temporary fix, IDs will be pulled from json later
             tmp.appearanceData.iNPC_ID = i;
             i++;
-            
+
             NPCs[tmp.appearanceData.iNPC_ID] = tmp;
         }
```

Comment:
We moved this code into `TableData.cpp` while refactoring. You can solve the merge conflict by removing it from here.